dipy/io/image.py

```diff
@@ -24,3 +25,44 @@ def load_nifti(fname, return_img=False, return_voxsize=False,
 def save_nifti(fname, data, affine, hdr=None):
     result_img = nib.Nifti1Image(data, affine, header=hdr)
     result_img.to_filename(fname)
+
+
+def load_affine_matrix(fname):
+    """
+        Parameters
+        ---------
+        fname : str
+            The file containing the saved affine matrix.
+    """
+    return np.loadtxt(fname)
+
+
+def save_affine_matrix(fname, affine):
+    """
+    Parameters
+    ---------
+    fname : string
+        File name to save the affine matrix.
+    affine : numpy array
+        The object containing the affine matrix.
+    """
+    np.savetxt(fname, affine)
+
+
+def save_quality_assur_metric(fname, xopt, fopt):
+    """
```

Comment:
you should say that the output is text and back parsable because you add `fopt` at the end of the file...